04/app.js

```diff
@@ -0,0 +1,25 @@
+const arr = createArr(10, 200);
+console.log(arr);
+const largest = getLargest(arr);
+console.log(largest);
+const average = getAverage(largest);
+console.log(average);
+
+
+function createArr(min, max) {
+    let arr = [];
+    for(let i = 0; i < 20; i++) {
+        arr.push(Math.round(Math.random() * (max-min) + min));
+    }
+    return arr;
+}
+
+function getLargest(arr) {
+    return arr.sort((a, b) => b - a).slice(0, 10);
+}
+
+function getAverage(arr) {
+    if (arr.length === 0) return null;
+    const sum = arr.reduce((a, b) => a+b);
```

Comment:
Do `reduce()` warto dodać drugi parametr jako 0 tj. `reduce(function() { /*...*/ }, 0)`, aby wykluczyć błąd przy pustej tablicy :)

Reply:
nawet jeśli warunek linijkę wyżej sprawia, że pusta tablica na pewno nie zostanie przekazana?

Reply:
Tak na prawdę to umknął mi ten warunek :D ale zastanawiam się czy nie lepiej zwrócić `0`.
Pozostawiam Tobie decyzje - oba rozwiązania będą ok :)